database/couchdb.js

```diff
@@ -0,0 +1,44 @@
+const dotenv = require('dotenv');
+dotenv.config();
+const nano = require('nano')(`http://admin:${process.env.COUCH_DB_PASSWORD}@localhost:5984`);
+const faker = require('faker');
+const { config } = require('../config.js');
+const { listPhotos }  = require('./aws-s3.js');
+
+const seed = async (database) => {
+
+  const photos = await listPhotos();
+  const docs = [];
+
+  for (let i = 0; i < 998; i++) {
```

Comment:
Why are you running this for loop until i is 998? Where did you get that number from?

Reply:
I had just tested with 2, so filled in the rest with 998